src/coverlet.core/CoverageDetails.cs

```diff
@@ -6,6 +6,7 @@ public class CoverageDetails
     {
         public double Covered { get; internal set; }
         public int Total { get; internal set; }
+        public double AverageModulePercent { get; internal set; }
```

Comment:
I agree with u, at first try I wrote with this approach, just more verbose than `ToString("0.00")`, but as u said we have different outputs if only use string format:
> Ex: threshold value is "85.25" and lets say line value is "85.2463", then if condition will evaluate to true and we output coverage failed for average threshold. But while displaying, we print "85.25" because we do ToString("0.00") which will do normal rounding.

I realized about it, but I needed another oint of view, tks! I'll work on it